soh/soh/SaveManager.cpp

```diff
@@ -19,7 +19,7 @@
 extern "C" SaveContext gSaveContext;
 
 void SaveManager::WriteSaveFile(const std::filesystem::path& savePath, const uintptr_t addr, void* dramAddr,
-                           const size_t size) {
+                                const size_t size) {
```

Comment:
It seems like a lot of whitespace was changed in this file, which makes it hard to tell what changes are relevant for your sectional functionality.  Is it possible to undo these formatting changes?

Reply:
I hate the default line length in the VS project generation...